bundles/AdminBundle/Resources/public/js/pimcore/object/quantityvalue/unitsettings.js

```diff
@@ -105,9 +105,9 @@ pimcore.object.quantityValue.unitsettings = Class.create({
 
         var typesColumns = [
             {flex: 1, dataIndex: 'id', text: t("id"), filter: 'string'},
-            {flex: 1, dataIndex: 'abbreviation', text: t("abbreviation"), editor: new Ext.form.TextField({listeners: {change: pimcore.helpers.htmlEncodeTextField}}), filter: 'string'},
-            {flex: 2, dataIndex: 'longname', text: t("longname"), editor: new Ext.form.TextField({listeners: {change: pimcore.helpers.htmlEncodeTextField}}), filter: 'string'},
-            {flex: 1, dataIndex: 'group', text: t("group"), editor: new Ext.form.TextField({listeners: {change: pimcore.helpers.htmlEncodeTextField}}), filter: 'string', hidden: true},
+            {flex: 1, dataIndex: 'abbreviation', text: t("abbreviation"), editor: new Ext.form.TextField(), filter: 'string'},
```

Comment:
we don't need `renderer: Ext.util.Format.htmlEncode` here? 
Also all the following fields?

Reply:
no since its set as default a few lines below :) 

https://github.com/pimcore/pimcore/blob/26a76f97ecac56645b96720b40d0d565598b0e33/bundles/AdminBundle/Resources/public/js/pimcore/object/quantityvalue/unitsettings.js#L201-L203